Doc/library/importlib.metadata.rst

```diff
@@ -127,7 +128,7 @@ a collection of all ``EntryPoint`` objects with ``names`` and ``groups``
 attributes for convenience::
 
     >>> sorted(eps.groups)  # doctest: +SKIP
-    ['console_scripts', 'distutils.commands', 'distutils.setup_keywords', 'egg_info.writers', 'setuptools.installation']
+    {'array_api', 'console_scripts', 'gui_scripts', 'pytest11', 'sphinx.html_themes'}
```

Comment:
The change to set here is incorrect. The command above is using `sorted()`, which returns a list.